README.md

```diff
@@ -65,11 +65,48 @@ TODO_BOARD_ID
 ### gunicorn support
 Support for gunicorn local run 
 ```
-poetry run gunicorn --daemon --bind 0.0.0.0:5000 -w 1 "wsgi:create_app()"
+$ poetry run gunicorn --daemon --bind 0.0.0.0:5000 -w 1 "wsgi:create_app()"
 ```
 
 ### Vagrant
 You can also use vagrant
 ```
 $ vagrant up
+```
+
+### Docker Compose
+
+```
+$ docker-compose build && docker-compose up -d <target>
+```
+
+
+### Docker
+
+Build the container
+```
+$ docker build --tag todo-app .
+```
+
+Run the container on port 5000
+```
+$ docker run -d --name todo-app --env-file=.env -p 5000:5000 todo-app
+```
+
+Building and running development 
+```
+$ docker build --target development --tag todo-app:dev .
+$ docker run --env-file ./.env -p 5100:5000 --mount type=bind,source="$(pwd)/todoapp",target=/app/todoapp todo-app:dev
+```
+
+Building and running production 
+```
+$ docker build --target production --tag todo-app:prod .
+$ docker run --env-file ./.env -p 5200:5000 todo-app:prod
+```
+
+Building and running tests 
+```
+$ docker build --target runtests --tag todo-app:runtests .
+$ docker run --env-file ./.env.test -p 5300:5000 todo-app:runtests
```

Comment:
Why are you publishing a port when just running tests?

Also, your unit/integration tests should use .env.test but your selenium tests make a real board and so need values from the real .env

This means you will need to run them in a separate command. Group your tests into two folders so you can run:
`docker run --env-file .env todo-app:runtests tests`
`docker run todo-app:runtests tests`